src/cookbooks/operational_monitoring.md

```diff
@@ -0,0 +1,120 @@
+  ...
+}
+```
+
+Note that each key in `dimensions.json` is the name of a dimension that will appear as a dropdown in the generated dashboards and can be referenced in the `<project_name>.json` file below. For each dimension, the following fields exist:
+
+- `source`: The BigQuery table name to be queried.
+- `sql`: The SQL used in the `select` clause for this probe.
+
+#### `probes.json`
+
+```
+{
+  "CONTENT_SHUTDOWN_CRASHES": {
+    "source": "mozdata.telemetry.crash",
+    "category": "stability",
+    "type": "scalar",
+    "sql": "IF(REGEXP_CONTAINS(payload.process_type, 'content') AND REGEXP_CONTAINS(payload.metadata.ipc_channel_error, 'ShutDownKill'), 1, 0)"
+  },
+  "CONTENT_PROCESS_COUNT": {
+    "source": "mozdata.telemetry.main_nightly",
+    "category": "performance",
+    "type": "histogram",
+    "sql": "payload.histograms.content_process_count"
+  },
+ ...
+}
+```
+
+Note that each key in `probes.json` is the name of a probe that will be used in the generated dashboards and can be referenced in the `<project_name>.json` file below. For each probe, the following fields exist:
+
+- `source`: The BigQuery table name to be queried.
+- `category`: This can be any string value. It's currently not being used but in the future, this could be used to visually group different probes by category.
+- `type`: This is used to determine the method of aggregation to be applied.
+- `sql`: The SQL used in the `select` clause for this probe.
+
+#### `<project_name>.json`
+
+```
+{
+  "name": "Fission Release Rollout",
+  "slug": "bug-1732206-rollout-fission-release-rollout-release-94-95",
+  "channel": "release",
+  "boolean_pref": "environment.settings.fission_enabled",
+  "xaxis": "submission_date",
+  "start_date": "2021-11-09",
+  "analysis": [
+    {
+      "source": "mozdata.telemetry.main_1pct",
+      "dimensions": ["cores_count", "os"],
+      "probes": [
+        "CONTENT_PROCESS_COUNT",
+      ]
+    }, {
+      "source": "mozdata.telemetry.crash",
+      "dimensions": ["cores_count", "os"],
+      "probes": [
+        "CONTENT_SHUTDOWN_CRASHES",
+      ]
+    }
+  ]
+}
+```
+
+- `name`: Name that will be used as the generated Looker dashboard title.
+- `slug`: The slug associated with the rollout that is being monitored.
+- `channel`: `release`, `beta`, or `nightly`. The channel the rollout is running in.
+- `boolean_pref`: A SQL snippet that results in a boolean representing whether a user is included in the rollout or not (note: if this is not included, the slug is used to check instead).
+- `xaxis`: `submission_date` or `build_id`. specifies the type of monitoring desired as described above.
+- `start_date`: `yyyy-mm-dd`, specifies the oldest submission date or build id to be processed (where build id is cast to date). If not set, defaults to the previous 30 days.
+- `analysis`: An array of objects with the fields `source`, `dimensions`, and `probes` where each object represents data that will be pulled out from a different source.
```

Comment:
Does each object here turn into one tile on the dashboard?
I think that's the thing that's missing to me right now: how does this list correspond to what I see in the end.

Can you link me to the json definitions for the existing Fission dashboard? Then I might be able to better understand this.

Reply:
[Here](https://console.cloud.google.com/storage/browser/operational_monitoring;tab=objects?forceOnBucketsSortingFiltering=false&project=emtwo-252813&prefix=&forceOnObjectsSortingFiltering=false) is the bucket with the fission json files